business-registry-dashboard/app/components/Modal/ManageBusiness.vue

```diff
@@ -0,0 +1,173 @@
+<script setup lang="ts">
+import type { AccordionItem } from '#ui/types'
+import { FetchError } from 'ofetch'
+import { StatusCodes } from 'http-status-codes'
+const brdModal = useBrdModals()
+const { t } = useI18n()
+const { $authApi } = useNuxtApp()
+
+const props = defineProps<{
+  business: ManageBusinessEvent
+}>()
+
+const loading = ref(true)
+const hasBusinessAuthentication = ref(false)
+const contactEmail = ref('')
+const affiliatedAccounts = ref<Array<{branchName?: string, name: string, uuid: string }>>([])
+
+const businessDetails = computed(() => ({
+  isFirm: props.business.legalType === CorpTypes.SOLE_PROP || props.business.legalType === CorpTypes.PARTNERSHIP,
+  isCorporation: props.business.legalType === CorpTypes.BC_COMPANY,
+  isBenefit: props.business.legalType === CorpTypes.BENEFIT_COMPANY,
+  isCorpOrBenOrCoop: (props.business.legalType === CorpTypes.BC_COMPANY || props.business.legalType === CorpTypes.BENEFIT_COMPANY || props.business.legalType === CorpTypes.COOP) && contactEmail.value !== '',
+  isCoop: props.business.identifier.toUpperCase().startsWith('CP'),
+  name: props.business.name,
+  identifier: props.business.identifier
+}))
+
+const showEmailOption = computed(() => {
+  return (businessDetails.value.isCorpOrBenOrCoop || businessDetails.value.isFirm) && contactEmail.value !== ''
+})
+
+const enableDelegationFeature = computed(() => {
+  // return LaunchDarklyService.getFlag(LDFlags.EnableAffiliationDelegation) || false // TODO: fix after adding launch darkly
+  return true
+})
+
+const showPasscodeOption = computed(() => {
+  // const allowableBusinessPasscodeTypes: string = LaunchDarklyService.getFlag(LDFlags.AllowableBusinessPasscodeTypes) || 'BC,SP,GP' // TODO: implememnt after adding launch darkly
+  const allowableBusinessPasscodeTypes: string = 'BC,SP,GP'
```

Comment:
I'm guessing we probably don't need to indicate here the string type (`:string = `)?

Reply:
For these simpler objects the type is inferred pretty well, maybe when I add in the launchdarkly stuff I'll need to update it.